apps/web/src/lib/supabase/mock.ts

```
@@ -0,0 +1,74 @@
+      },
+      select: async () => {
+        return {
+          data: [],
+          error: null,
+        };
+      },
+    };
+  }
+
+  auth = {
+    onAuthStateChange: (callback: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)